src/assets/Zombie.java

```diff
@@ -1,245 +1,244 @@
-package assets;
-
```

Comment:
Why was this removed?

Reply:
A Merge Error when I was trying to update this branch. I corrected it but the changes still show up.

Reply:
Same goes for levelloader